### PR TITLE
chore: run "make generate code/fix" to update generated code and fix linting

### PR DESCRIPTION
**Pull request by machi1990**
## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA have been completed
- [ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)
- [ ] Documentation added for the feature
- [ ] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer
- [ ] Required metrics/dashboards/alerts have been added (or PR created).
- [ ] Required Standard Operating Procedure (SOP) is added.
- [ ] JIRA has created for changes required on the client side

### CODE REVIEW

File: internal/kafka/internal/handlers/service_accounts.go [hunk 1]
```diff
@@ -173,14 +173,14 @@ func (s serviceAccountsHandler) GetServiceAccountById(w http.ResponseWriter, r *
 	handlers.HandleGet(w, r, cfg)
 }
 
-func (s serviceAccountsHandler) GetSsoProvider(w http.ResponseWriter, r *http.Request){
+func (s serviceAccountsHandler) GetSsoProvider(w http.ResponseWriter, r *http.Request) {
```
Comment (akoserwal) [inline internal/kafka/internal/handlers/service_accounts.go:176]: should we also make `GetSsoProvider` to `GetSsoProviders`?

**Review by akoserwal — COMMENTED** [review]
LGTM, one minor comment

File: internal/kafka/internal/handlers/service_accounts.go (hunk 1 above)
Comment (machi1990) [reply internal/kafka/internal/handlers/service_accounts.go:176]: This code i.e the method name is not generated that's why it wasn't changed. Let me know if that's the change you'd want to make also and I can modify it.

Comment (akoserwal) [reply internal/kafka/internal/handlers/service_accounts.go:176]: we can keep that way or wdyt?

Comment (machi1990) [reply internal/kafka/internal/handlers/service_accounts.go:176]: Thanks for raising this. So I checked how are other handlers named and they seem to align with the `operationId` defined in the OpenAPI file. Let's change it and add the `s` too.

**Comment by machi1990** [comment]
> LGTM, one minor comment

@akoserwal thanks for the review. I've update the handlers method name. Can you have a look again?

**Review by akoserwal — APPROVED** [review]

**Comment by akoserwal** [comment]
> > LGTM, one minor comment
> 
> @akoserwal thanks for the review. I've update the handlers method name. Can you have a look again?

thank you so much

**Comment by machi1990** [comment]
@dhirajsb @valdar @lburgazzoli can you have a look at the connectors change? It's mainly after the run of `make code/fix`. Thanks

**Comment by machi1990** [comment]
> @dhirajsb @valdar @lburgazzoli can you have a look at the connectors change? It's mainly after the run of `make code/fix`. Thanks

We'll have to get this in. The change in the connectors code were only stylistic and after `make code/fix` was run.

**Review by valdar — APPROVED** [review]
Lgtm

**Pull request merged**